test/Staking.t.sol

```diff
@@ -17,7 +18,7 @@ contract BlueberryStakingTest is Test {
 
     IERC20 public mockUSDC;
 
-    address public treasury = address(0x1);
+    address public treasury = address(99);
```

Comment:
FYI, forge has a nice cheat called `makeAddr()` which generates an address from a string name. You can use this to ensure tests addresses are deconflicted without manually choosing a random number for each one.

https://book.getfoundry.sh/reference/forge-std/make-addr

Reply:
I normally use `makeAddr` when developing tests but kept with the `address` to continue with convention from the previous developers code. I agree that its better to generate an address from a string name, so I'll go through and clean up the rest of the instances.